app/test_main.py

```diff
@@ -1,3 +1,31 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+def test_raising_error_correctly() -> None:
+    with pytest.raises(TypeError):
+        get_human_age("1", "2")
```

Comment:
Please add more cases with different data types. Use parametrize decorator for it